src/commands/environment/backupRestore/README.md

```diff
@@ -216,9 +228,10 @@ await restoreEnvironment(params);
 
 - **Scheduled Publishing Times**: The current API format doesn't support inclusion of the publishing time for variants scheduled to be published. The tool instead places scheduled variants into the draft step (the first step in the workflow).
 
-### Asset Size
+### Asset Limitations
 
 - **Asset Size Limit**: The Management API accepts only assets smaller than 100 MB. If your backup file contains assets larger than that (they can be uploaded through the UI), the tool won't be able to import them.
+- **Asset Quality**: Assets fetched by the tool may be compressed and metadata can be missing. You may find more information about asset compression in the [quality](https://kontent.ai/learn/docs/apis/image-transformation-api#a-quality-parameter) and [lossles](https://kontent.ai/learn/docs/apis/image-transformation-api#a-lossless-parameter) parameter documentation.
```

Comment:
lossles only goes for the images with webp format. Can you please remove the part mentioning it? I think that the quality parameter is enough for reference